src/Controller/ComponentRegistry.php

```diff
@@ -61,6 +58,20 @@ public function getController()
     }
 
     /**
+     * Set the controller associated with the collection.
+     *
+     * @param Controller $Controller Controller instance.
+     * @return void
+     */
+    public function setController(Controller $Controller = null)
```

Comment:
@lorenzo https://github.com/cakephp/cakephp/pull/7301/files#diff-eb1df6e6abe39034b62ef18fce423e33R45 here it's called `$Controller`. So should I change it to `$controller` everywhere in my changes?